editor.planx.uk/src/@planx/components/Checklist/Public/components/ChecklistItems.tsx

```diff
@@ -0,0 +1,56 @@
+import Grid from "@mui/material/Grid";
+import { Option } from "@planx/components/shared";
+import ImageButton from "@planx/components/shared/Buttons/ImageButton";
+import { FormikProps } from "formik";
+import React from "react";
+import FormWrapper from "ui/public/FormWrapper";
+import ChecklistItem from "ui/shared/ChecklistItem/ChecklistItem";
+
+import { ChecklistLayout } from "./VisibleChecklist";
+
+interface NonExclusiveChecklistItemsProps {
+  nonExclusiveOptions: Option[];
+  layout: ChecklistLayout;
+  changeCheckbox: (id: string) => () => void;
+  formik: FormikProps<{ checked: Array<string> }>;
+  exclusiveOptionIsChecked: boolean;
+}
+
+export const NonExclusiveChecklistItems = ({
```

Comment:
As per the file name, I actually think that just `ChecklistItems` might be better here.

The combination of `ChecklistItems` and `ExclusiveChecklistItem` still feels meaningful to me.